include/taskolib/Sequence.h

```diff
@@ -567,6 +590,8 @@ class Sequence
 
     std::string label_; ///< Sequence label.
 
+    std::string maintainers_{""}; ///< One or more maintainers.
```

Comment:
```suggestion
    std::string maintainers_; ///< One or more maintainers.
```
The empty initializer just adds visual noise, nothing else.